src/docs/product/sentry-basics/search/searchable-properties.mdx

```diff
@@ -144,34 +143,33 @@ Release properties overlap with event and issue search properties, so they don't
+| measurements.<br></br>stall_percentage | Stall percentage is equal to the `stall_total_time` divided by the `transaction.duration`. Only applies to React Native. | x |  |  | number |
+| measurements.<br></br>stall_total_time | The [total stall time](/platforms/react-native/performance/instrumentation/automatic-instrumentation/#stall-tracking) is the total combined time, in milliseconds, of all stalls. Only applies to React Native. | x |  |  | duration |
+| measurements.ttfb | [Time To First Byte (TTFB)](/product/performance/web-vitals/#time-to-first-byte-ttfb) measures the time that it takes for a user's browser to receive the first byte of page content. | x |  |  | duration |
+| measurements.<br></br>ttfb.requesttime | The time between start of the request and start of the response (see [diagram](https://web.dev/ttfb/#what-is-ttfb)). | x |  |  | duration |
 | message | Returns errors with the matching message or transactions with matching transaction name. Also matches on any message containing the supplied value.<br></br><br></br>Searching `message:undefined` will match an event with a message of `undefined is not an object`.<br></br><br></br>Raw text searches (searches without the `message` key) are also checked against this field. For errors, the message can be a concatenatenation of elements, so searches might include unexpected results. | x | x |  | string |
-| min(numeric field) | Returns results with a matching minimum value for the field entered. | x |  |  | matches field |
+| min(numeric field) | Returns results with a matching minimum value for the field entered. | x |  | x | matches field |
 | os.build | The internal build revision of the operating system. | x | x |  | string |
-| os.name | TBA |  |  | x | TBA |
 | os.kernel_version | The independent kernel version string. This is typically the entire output of the `uname` syscall. | x | x |  | string |
-| percentile(field,level) | Returns results with an approximate percentile of the field to the level. The level can be between `0` and `1`. For example, if you wanted to find the 50th percentile of transaction durations, you would enter `percentile(transaction.duration, 0.5)`. | x |  |  | number |
-| platform | Name of the platform. This is only a metrics property for [valid platforms](https://github.com/getsentry/relay/blob/9d0d2dc54233d46043a369e1dacf7db900953c14/relay-general/src/protocol/constants.rs#L1-L21), defaulting to `other`. | x |  | x | string |
+| percentile(field, level) | Returns results with an approximate percentile of the field to the level. The level can be between `0` and `1`. For example, if you wanted to find the 50th percentile of transaction durations, you would enter `percentile(transaction.duration, 0.5)`. | x |  | x | number |
```

Comment:
So we actually don't include a space on any of the entries where we have multiple words in the parentheses. That's the way these were entered in the list that we worked from in Notion. I assume it's because in the application, there's no space there and if you were writing that query (though obviously it auto-populates for you), you wouldn't include a space. I'll defer to you on this though. Do what you think makes sense.